src/gui/sdlmain.cpp

```diff
@@ -3156,6 +3185,12 @@ void Config_Add_SDL() {
 	        "             WxH format. For example: 1024x768.\n"
 	        "             Scaling is not performed for output=surface.");
 
+	pstring = sdl_sec->Add_string("windowposition", always, "auto");
+	pstring->Set_help("Set initial window position when running in windowed mode:\n"
+	                  "  auto:      Let the window manager decide the position.\n"
+	                  "  <custom>:  Set window position in X,Y format.\n"
+	                  "             For example: 250,100");
+
```

Comment:
Suggest a bit more help to the user indicating where these coordinates push-in from. For example, is 0,0 the top-left corner of the monitor or the bottom right?

Reply:
Sure, will add more clarification. I wanted to keep it succint, but maybe it's a bit too much so :)